mission_control/navigator_missions/navigator_singleton/navigator.py

```diff
@@ -222,10 +221,10 @@ def catch_error(self, failure):
             # Handle error
 
     @util.cancellableInlineCallbacks
-    def start_search(self, timeout=60, loop=True, spotings_req=2, speed=0.1):
+    def start_search(self, timeout=60, loop=True, spotings_req=2, **kwargs):
```

Comment:
timeout is unused here, is it supposed to be? Just checking that it's not supposed to propagate through `**kwargs`